### PR TITLE
Map several simple potato entities and blocks.

### PR DESCRIPTION
**Pull request by CallMeEchoCodes**
First yarn contribution! I just went in and did a bunch of simple ones so you smart people don't need to waste time on them.

### CODE REVIEW

File: mappings/net/minecraft/render/entity/feature/PoisonousPolytryaFeatureRenderer.mapping [hunk 1]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_9623 net/minecraft/render/entity/feature/PoisonousPolytryaFeatureRenderer
```
Comment (modmuss50) [inline mappings/net/minecraft/render/entity/feature/PoisonousPolytryaFeatureRenderer.mapping]: Spelling seems incorrect here.`Polytra`

**Comment by modmuss50** [comment]
Hi, many thanks for this PR, sadly it seems to have a lot of overlap with #3831 from @Shnupbups, I did a quick spot check and it seems most of the names are identical so thats great 👍

Im not sure what the best way forward is, it might just be easier to merge #3831. I dont like to close PRs especially when there is nothing wrong with it and it would normally be fine to merge, #3831 was first so this seems fair to me. Sorry about this, I hope you understand.

**Comment by Shnupbups** [comment]
Yeah, sorry about this, but I'm gonna close this PR as everything was covered by mine. Don't be discouraged though, you did a good job!

**Comment by CallMeEchoCodes** [comment]
np! gonna be honest i didnt even check other prs before i made this so its kinda my fault

**Pull request closed without merging**